lib/fog/aws/requests/dynamodb/update_item.rb

```diff
@@ -6,39 +6,33 @@ class Real
         #
         # ==== Parameters
         # * 'table_name'<~String> - name of table for item
-        # * 'key'<~Hash>:
-        #   * 'HashKeyElement'<~Hash>: info for primary key
-        #     * 'AttributeName'<~String> - name of attribute
-        #     * 'AttributeType'<~String> - type of attribute
-        #   * 'RangeKeyElement'<~Hash>: optional, info for range key
-        #     * 'AttributeName'<~String> - name of attribute
-        #     * 'AttributeType'<~String> - type of attribute
-        # * 'attribute_updates'<~Hash>:
-        #   * 'AttributeName'<~String> - Attribute to update
-        #     * 'Value'<~Hash> - formated as {type => value}
-        #     * 'Action'<~String> - action to take if expects matches, in %w{ADD DELETE PUT}, defaults to PUT
+        # * 'key'<~Hash> - list of elements to be updated and their value
+        #   {
+        #     "ForumName": {"S": "Amazon DynamoDB"},
+        #     "Subject": {"S": "Maximum number of items?"}
+        #   }
+        #
         # * 'options'<~Hash>:
-        #   * 'Expected'<~Hash>: data to check against
-        #     * 'AttributeName'<~String> - name of attribute
-        #     * 'Value'<~Hash> - a value to check for the value of
-        #     or
-        #     * 'Exists'<~Boolean> - set as false to only allow update if attribute doesn't exist
+        #   * 'KeyConditionExpression'<~String> - the condition elements need to match
+        #   * 'ExpressionAttributeValues'<~Hash> - values to be used in the key condition expression
         #   * 'ReturnValues'<~String> - data to return in %w{ALL_NEW ALL_OLD NONE UPDATED_NEW UPDATED_OLD}, defaults to NONE
         #
         # ==== Returns
         # * response<~Excon::Response>:
         #   * body<~Hash>:
         #     varies based on ReturnValues param, see: http://docs.amazonwebservices.com/amazondynamodb/latest/developerguide/API_UpdateItem.html
-        def update_item(table_name, key, attribute_updates, options = {})
+        #
+        # See DynamoDB Documentation: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html
+        #
+        def update_item(table_name, key, options = {})
```

Comment:
Returned object changes should probably be ok (aws tends to do this in an additive way). It is less of a break anyway.